common/domain/src/main/kotlin/no/nav/mulighetsrommet/domain/dbo/HistorikkDbo.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.mulighetsrommet.domain.dbo
+
+import kotlinx.serialization.Polymorphic
+import kotlinx.serialization.Serializable
+import no.nav.mulighetsrommet.domain.dto.Deltakerstatus
+import no.nav.mulighetsrommet.domain.serializers.LocalDateTimeSerializer
+import no.nav.mulighetsrommet.domain.serializers.UUIDSerializer
+import java.time.LocalDateTime
+import java.util.*
+
+@Serializable
+@Polymorphic
+sealed class HistorikkDbo {
```

Comment:
Hadde det gitt mening om vi kalte dette for noe litt mer spesifikt, f.eks. `tiltaksistorikk` over hele linja? Tenker da på databasetabell, dbo, repository osv.